### DISCUSSION TITLE
Spread Operator syntax (...) support for safari IOS 11 and 10

### DISCUSSION THREAD

**Discussion by judevs** [alpinejs/alpine#3568] [1. Help] [answered]
I'm using `alpinejs_3.12.0` in our landing page with tailwindcss, and when we are checking the browser compatibility, in safari IOS 11 and lower version or at least in safari IOS 10, we found this error and causing the site to break (used browserstack to check compatibility)
```
SyntaxError: Unexpected token '...'. Expected a property name.(anonymous function) @ alpinejs_3.12.0.min.js:1
```
Seems like the spread operator is not supported well in IOS 11 and 10 that's why it keeps throwing the error, does AlpineJS doesn't support safari IOS 11 and 10? 

Is there a work around here? minimum of support for older browser is safari IOS 10 and higher. 

Methods I tried.
- transpiling alpinejs core file to es5 (doesn't work - breaks the whole site)
- polyfill for spread operator, seems like there's no polyfill for spread oprator

**Comment by ekwoka** [comment, marked as answer]
Alpine requires ES6, that's the oldest you can safely transpile to because of async functions and proxies.

**Reply by judevs** [reply]
Ohhhhh I see, thank you

but if you know any work around for this, please let me know, my only problem is for the support of spread operator syntax.
